packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -278,6 +278,14 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
             'as a multiplier of the default timeout (e.g. "2x"), '
             'or as the string "none" to disable the timeout entirely.',
       )
+      ..addFlag(
+        'ignore-timeouts',
+        help:
+            'Ignore all timeouts. Useful when testing a big application '
+            'that requires a longer time to compile (e.g. running integration '
+            'tests for a Flutter app).',
```

Comment:
I'm not so happy with the helper text, as I assume it's confusing that the `timeout` help text says 'or as the string "none" to disable the timeout entirely' and a `--ignore-timeouts` flag, which sounds like the same thing but is not.

Reply:
I think it might be worth also updating the help text for `timeout` to explicitly state that it only applies to the timeout for individual tests. Maybe something like this:

```
The default timeout for individual tests, specified either in seconds (e.g. "60s"), as a multiplier of the default test timeout (e.g. "2x"), or as the string "none" to disable test timeouts entirely. This value does not apply to the default test suite loading timeout.
```